### ISSUE TITLE
Watch app no longer installing

### ISSUE THREAD

**Issue by ntompson** [home-assistant/home-assistant-iOS#450]
After having successfully used the 2.0.0 app on iPhone and watch for a while, in figuring out the #445 issue, I completely removed and reinstalled the app through TestFlight. I've got the iPhone app working OK, but I can't get the watch app to install. 

I've tried to install the watch app by hitting "Install" in the Apple Watch app on my phone (the Install button turns into a spinning circle which never stops spinning and never reports an error). I've also tried toggling the "Show App on Apple Watch" slider in TestFlight (which tries for a while and eventually provides a failure error message).

I've tried hard / soft reseting both my watch and my iPhone. 

The only thing I can think is that this is the first time I've attempted to install the iPhone / watch app after upgrading to iOS 13 / watchOS 6.

**Comment by ntompson** [comment]
Well I don't know. After attempting to install the watch app as described in my previous notes unsuccessfully multiple times, the watch app just turned up this morning. I have no idea how it got there or what the successful combination of actions was.

So is this an issue? I don't know. If no-one else has issues with iOS13 / watchOS 6, then it probably isn't. Let's see.

**Comment by TomBrien** [comment]
Watch apps can sometimes take a long time to install if there are several to do. Can we close this for now?

**Comment by ntompson** [comment]
Yes I think it's fair to close. I suspect the issue is more to do with TestFlight in iOS 13 than any problem with HA companion. Following the recent release of build 68, I did a full remove and reinstallation. This time I found that TestFlight refused to reinstall the HA companion app at all (endless spinning circle in the TestFlight app after I pressed install). I had to do a hard reset of my phone to get it to install. But then it was all fine. Watch app went in at the same time as the phone app.

So I am left thinking there is something shonky going on with TestFlight in iOS 13.
